src/components/HomeMain.component/HomeTrendingCard.component/home-trending-card.css

```diff
@@ -0,0 +1,17 @@
+
+
+:root {
+    --color-white: #ffffff;
```

Comment:
I will suggest you to make a common file for all such colors and all and import it in the main index.html file